autogpt_platform/backend/backend/util/settings.py

```diff
@@ -117,6 +117,16 @@ class Config(UpdateTrackingModel["Config"], BaseSettings):
         description="The port for agent server daemon to run on",
     )
 
+    database_api_host: str = Field(
+        default="0.0.0.0",
+        description="The host for database server API to run on",
+    )
+
+    database_api_port: int = Field(
```

Comment:
this needs to be added to docker compose and helm values

Reply:
hmm other host & ports were not added in the docker compose and it uses the default value too.